man/src/version.md

```diff
@@ -1,37 +1,57 @@
-This command displays version information for both the Docker client and 
-daemon. 
+The version command prints the current version number for all independently
+versioned Docker components.
 
 # EXAMPLES
 
 ## Display Docker version information
 
-The default output:
+The default output renders all version information divided into two sections;
+the "Client" section contains information about the Docker CLI and client
```

Comment:
Mostly aligning this content more with the online reference (although there's parts not yet included in the man-pages. I left that for a later exercise).